### PR TITLE
README update

### PR DESCRIPTION
**Pull request by ams**
@shrzen Hey! Do you want to merge this back into LM-3/usim?

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -257,3 +252,7 @@ answered lots of questions.  Steve helped me locate missing pages from
 emulators.  Bjorn has used the code, offered many suggestions, fixes
 and improvements.  And John's office is where I first saw a 3600
 console and said, "what's that?".
+
```
Comment (ams) [inline README]: Useless newlines?

File: README [hunk 2]
```diff
@@ -78,18 +78,18 @@ host where usim is running is called "SERVER" (Chaos address: 0404).
 
 Once your CADR is up and running running, in the listener type:
 
-  (si:set-sys-host "server" ':unix 0404) 
+  (si:set-sys-host "server" ':unix 0404)
```
Comment (ams) [inline README]: Useless space at end of line?

File: README [hunk 3]
```diff
@@ -64,7 +64,7 @@ Here are some interesting keybindings to try in the Listener, see the
   Terminal-0-M			; Disable **MORE** processing.
 
 For more details see the Lisp Machine Manual
-(https://lm-3.github.io/lmman.html).
+(https://lm-3.github.io/chinual.html).
```
Comment (ams) [inline README]: Yay!

**Comment by shrzen** [comment]
That would be nice.

**Pull request merged**